Extension/chain/dist/ChainNames.d.ts

```diff
@@ -0,0 +1,9 @@
+export declare const ChainNames: {
```

Comment:
These `dist` are "output" builds so have to gitignore (idk why it's not working 😅)